homeassistant/components/modem_callerid/config_flow.py

```diff
@@ -0,0 +1,65 @@
+"""Config flow for Modem Caller ID integration."""
+from phone_modem import PhoneModem, exceptions
+import serial
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_DEVICE
+
+from . import _LOGGER
+from .const import DEFAULT_DEVICE, DEFAULT_NAME
+from .const import DOMAIN  # pylint:disable=unused-import
+
+DATA_SCHEMA = vol.Schema({"name": str, "device": str})
+
+
+class PhoneModemFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Phone Modem."""
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initiated by the user."""
+        errors = {}
+
+        if user_input is not None:
+            device = user_input[CONF_DEVICE]
+
+            await self.async_set_unique_id(device)
+            self._abort_if_unique_id_configured()
+            try:
+                api = PhoneModem()
+                await api.test(device)
+
+            except (
+                FileNotFoundError,
+                exceptions.SerialError,
+                serial.SerialException,
+                serial.serialutil.SerialException,
+            ):
```

Comment:
Make the tuple of exceptions a constant so you can reuse it everywhere its needed.